pkgs/games/overgrowth/default.nix

```
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, makeDesktopItem
+, libGL, libGLU, SDL2, SDL2_net, gtk2, glib, bzip2
+, openal, libogg, libvorbis, libjpeg, freetype
+, overgrowth-data ? "" }:
```

```suggestion
}:
```
This is not a package nixpkgs has and relying like this on an overlay is not allowed.

---

It is not (necessarily) a package but a path to the data.  Is there anyway to provide a build-time configuration, perhaps naming it less ambiguously?

---

I don't think there is a better way, so we should add a short comment.

What happens if that data is not supplied and we pass ``-DAUX_DATA=`` to cmake?

---

`AUX_DATA` is for the binary to know the path to the data, otherwise (e.g. empty or not found) it looks for the current work dir (i.e. what the desktop file does but in this case it's neither or both, so it's only there so people can copy it over user dir and edit to avoid rebuild the engine).